tests/lib/nested.sh

```diff
@@ -196,17 +196,15 @@ create_nested_core_vm(){
     mkdir -p "$WORK_DIR/image"
     if [ ! -f "$WORK_DIR/image/ubuntu-core.img" ]; then
         local UBUNTU_IMAGE
-
-        if ! snap list ubuntu-image; then
-            snap install ubuntu-image --classic
-        fi
         UBUNTU_IMAGE=/snap/bin/ubuntu-image
 
         # create ubuntu-core image
         local EXTRA_FUNDAMENTAL=""
         local EXTRA_SNAPS=""
-        if [ -d "${PWD}/extra-snaps" ] && [ "$(find "${PWD}/extra-snaps/" -type f -name "*.snap" | wc -l)" -gt 0 ]; then
-            EXTRA_SNAPS="--snap ${PWD}/extra-snaps/*.snap"
+        if [ -d "${PWD}/extra-snaps" ]; then
+            for mysnap in $(find "${PWD}/extra-snaps/" -type f -name "*.snap"); do
```

Comment:
Do we have subdirectories under `extra-snaps/`? Otherwise maybe we could iterate directly over `extra-snaps/*.snap` instead of using find. If find is necessary, we can use a while read loop to keep shellcheck happy (read -r -d '' and find with -print0 if we want it to be really robust?)

Reply:
I agree with @cmatsuoka  on that, we don't have subdirectories here, a simple `for mysnap in extra-snaps/*.snap` would work.

Reply:
done